### ISSUE TITLE
Nothing exported?

### ISSUE THREAD

**Issue by drone1** [hadijaveed/reactive-handlebars#1]
Thanks for the nice code here.

I installed via ```npm i --save reactive-handlebars``` and noticed that importing/requiring the package results in an empty object. It looks like you might need to do:

```module.exports = ReactiveHbs``` or ```export default ReactiveHbs``` at the end of src/reactive-handlebars.js.

**Comment by drone1** [comment]
Looks like it doesn't load jquery or handlebars or lodash via NPM install either. :)

**Comment by hadijaveed** [comment]
@drone1 yes I agree. Thanks for your feedback Happy to make those modifications in a day or two. Also, feel free to open a PR in-case you want to make some modifications
